src/main/config.ts

```diff
@@ -21,7 +21,9 @@ export const defaultConfiguration = {
   radioEffects: 'on' as RadioEffects,
   hardwareType: 0,
   radioGain: 0,
-  alwaysOnTop: 'never' as AlwaysOnTopMode
+  alwaysOnTop: 'never' as AlwaysOnTopMode,
+  showExpandedRx: false,
+  transparentMiniMode: false
```

Comment:
These need to be added to `src/shared/config.type.ts` as well.

Reply:
They should be in the interface already, 

https://github.com/pierr3/TrackAudio/blob/new-design/src/shared/config.type.ts#L23-L25

Let me know if I've missed something

Reply:
I'm just dumb, I hadn't finished my coffee yet 🙈